evthread_pthread.c

```diff
@@ -178,12 +179,24 @@ evthread_use_pthreads(void)
 		evthread_posix_cond_signal,
 		evthread_posix_cond_wait
 	};
+
+	if (pthread_mutexattr_init(&attr_default))
+		return -1;
+
 	/* Set ourselves up to get recursive locks. */
 	if (pthread_mutexattr_init(&attr_recursive))
 		return -1;
 	if (pthread_mutexattr_settype(&attr_recursive, PTHREAD_MUTEX_RECURSIVE))
 		return -1;
 
+#ifdef EVENT__ENABLE_POSIX_PRIORITY_INHERITANCE
```

Comment:
Hm, introducing yet another compile-time option is superfluous for this, let's introduce `evthread_use_pthreads_flags()` and add it as a flag?

Reply:
Sure, will do!